bot/cogs/snakes.py

```diff
@@ -40,9 +123,79 @@ def __init__(self, bot: AutoShardedBot):
         :param ctx: Context object passed from discord.py
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         """
+        embed = discord.Embed(color=0x3E885B)
+        if name and name.lower() == "python":
+            # handle Python special case
+            embed.add_field(
+                name="Python (programming language)",
+                value=(
+                    "*Guido van Rossum*\n\n"
+                    "This language is neither dangerous nor venomous and can be found in software globally"
+                ),
+                inline=False
+            )
+            embed.set_image(url=await self.get_snek_image("python programming language"))
+        else:
+            snek_info = await self.get_snek(name)
+            if not snek_info['pass']:
+                embed = discord.Embed(color=0xDD3D5A)
+                embed.add_field(
+                    name="We Couldn't Find That :snake:",
+                    value=f':x: "{name}" didn\'t match anything in our database :frowning:',
+                    inline=False
+                )
+                await ctx.channel.send(embed=embed)
+                return  # break out of rest of method
+            if snek_info['is_venomous']:
+                # if the snake is venomous -- use the fancy check icon
+                venom_info = f":white_check_mark: venomous\n\n"
+            else:
+                # if the snake is not venomous -- use the fancy not allowed icon
+                venom_info = f":no_entry_sign: NOT venomous\n\n"
+            additional_info = ''  # required to prevent referencing before assignment
+            if snek_info['matches_count'] and snek_info['matches_count'] > 1:
+                additional_info = f"\n\n" \
+                                  f"This search matched {snek_info['matches_count']} snakes. " \
+                                  f"Try creating a more specific query for information about a particular snake. " \
+                                  f"(This is a random selection from the {snek_info['matches_count']}.)"
+            embed.add_field(
+                name=titlecase(snek_info['common_name']),
+                value=(
+                    f":microscope: *{titlecase(snek_info['scientific_name'])}*\n\n"
+                    f"{venom_info}"
+                    f":globe_with_meridians: Found in {snek_info['locations']}"
+                    f"{additional_info}"
+                ),
+                inline=False
+            )
+            embed.set_image(url=snek_info['image_url'])
+        await ctx.channel.send(embed=embed)
 
     # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
 
+    @command(aliases=["f"])
+    async def fact(self, ctx: Context):
```

Comment:
I would just like to point out that you put a ton of work into the `get` command, and not much into your extra stuff. 

The marking is heavily weighted towards the extra creative stuff. But you still have 12 hours!